### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by Krzychumag** [Neurodynasoft/OCTAGON#1]
Good morning  Alain Bellon.

I sincerely apologize to ask if you could ask for sharing OCTAGON Chess Tournament GUI.

Best regards
Krzysztof

**Comment by OfekShochat** [comment]
Yeah I agree, but if he really wants to be sneaky with his code, he can compile it to webassembly and the python to exe using nuitka both in 10 seconds
